apps/crunchy-postgres/tests/node_affinity/test.yml

```diff
@@ -0,0 +1,100 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+    
+        - name: Generate the litmus result CR to reflect SOT (Start of Test) 
+          template: 
+            src: /litmus-result.j2
+            dest: litmus-result.yaml
+          vars: 
+            test: "{{ test_name }}"
+            app: ""
+            chaostype: ""
+            phase: in-progress
+            verdict: none
+    
+        - name: Apply the litmus result CR
+          shell: kubectl apply -f litmus-result.yaml
+          args:
+            executable: /bin/bash
+          register: lr_status 
+          failed_when: "lr_status.rc != 0"
+
+        - name: Getting Pod name of Application
+          shell: kubectl get pod -n {{ namespace }} -l {{ app_label }} -o jsonpath='{.items[0].metadata.name}'
+          register: pod_name
+          
+        - name: Getting replica-pod name
+          shell: kubectl get pod -n {{ namespace }} -l openebs.io/persistent-volume-claim={{ pvc_name }}-{{ pod_name.stdout }} -o jsonpath='{.items[1].metadata.name}'
```

Comment:
label has to be changed according to 0.7.

Reply:
@shashank855 Replica pod does not contain application pod traces in its name, then how are we getting the replica pod name here? Can you please explain?

Reply:
@dargasudarshan Replica **pod's label** contain the traces the of application pod name. We are getting the replica pod name using this label here.

Reply:
@shashank855 I have checked the same label does not have application pod traces in case of deployment?
Please see the snippet below. Can we make generic to work for both sts and deployments? Just a taught..
```
sudarshan_darga@cloudshell:~/test-tools/dd-client (strong-eon-153112)$ kubectl get pods pvc-f2be77f6-d05c-11e8-881b-42010a80026a-rep-885694bb9-dhgc7 -n app-percona-ns -o yaml
apiVersion: v1
kind: Pod
metadata:
  annotations:
    openebs.io/capacity: 5G
    openebs.io/storage-pool: default
  creationTimestamp: 2018-10-15T12:26:43Z
  generateName: pvc-f2be77f6-d05c-11e8-881b-42010a80026a-rep-885694bb9-
  labels:
    openebs.io/persistent-volume: pvc-f2be77f6-d05c-11e8-881b-42010a80026a
    openebs.io/persistent-volume-claim: percona-mysql-claim
```